Classes/Event/EventPublisher.php

```diff
@@ -104,7 +105,11 @@ public function publishMany(string $streamName, array $events, int $expectedVers
             $eventClassName = $this->eventTypeResolver->getEventClassNameByType($rawEvent->getType());
             $event = $this->propertyMapper->convert($rawEvent->getPayload(), $eventClassName, $configuration);
             foreach ($this->eventListenerLocator->getSynchronousListeners($rawEvent->getType()) as $listener) {
-                call_user_func($listener, $event, $rawEvent);
+                if ($listener[0] instanceof EventListenerInvocationInterface) {
```

Comment:
`$listener` is an instance of [callable](http://php.net/manual/en/language.types.callable.php). PHPs implementation of that is a bit ...weird.. but it could be a closure or an array ['<className>', '<methodName>'] for example